appointment/upload-documents/uploadDocumentsController.js

```diff
@@ -49,6 +48,37 @@ define('uploadDocumentsController', [], function() {
 		const ACCEPTABLE_FILE_TYPES = ['application/pdf', 'image/jpeg', 'image/png'];
 		const ACCEPTABLE_FILE_EXTENSIONS = ['.pdf', '.jpeg', '.jpg', '.png'];
 
+		$scope.storeConsent = function() {
+			const reviewConsent = $scope.consentData.reviewConsent;
+			const virtualConsent = $scope.consentData.virtualConsent;
+			const signature = $scope.consentData.signature;
```

Comment:
You should probably check these variables here again to make sure they're valid, and if not, maybe raise an error with NotificationUtilities, or set some variable and in the HTML show an invalid form box. It's better to avoid the BE call if we know the inputs are invalid on the FE

Reply:
That's a good thought. Would that just be repeating the ng-disable logic?

Reply:
Yep